### DISCUSSION TITLE
Amazon triple containing

### DISCUSSION THREAD

**Discussion by Lenny9789** [apache/incubator-mxnet#20025] [General]
Hi, I'm very glad to find you on this website. I'm a developer from China. Now I does working on Bluetooth Sigmesh . I plan to access the Amazon echo for my device , and then I recgnized before I need the BLE Mesh Credentials- A triple containing the OOB Type (Static only for now), OOB Data (pub key), and Device UUID.
But I has clicked entire website still connot find out the documentation for how to get my triple containing.
May I get some help form you , very appreciate.

**Comment by szha** [comment]
Hi @liubenyuan. From what you asked, it doesn't seem related to Apache MXNet. While Apache MXNet have many members working at Amazon, the project is not affiliated with the company. It's unclear how this project can help what you asked.
